modules/ROOT/pages/db-util.adoc

```diff
@@ -273,13 +144,11 @@ To do this, you use the `validate-that` operation. Set the `results` property to
 query="SELECT * FROM jobtitlelookup WHERE JOBTITLE='Developer';" returns="&quot;JOBTITLE&quot;,&quot;EECSALARYORHOURLY&quot;,&quot;JOBTITLEID&quot;\n&quot;Developer&quot;,&quot;10&quot;,&quot;DEV&quot;" />
 ----
 
-=== Executing SQL instructions and SQL queries
+=== Execute
 
-The MUnit DB Server allows you to execute instructions on the in-memory databases, so
-you can add or remove registries before a test, and also check if your data was
-stored correctly.
+The MUnit DB Server allows you to execute instructions on the in-memory databases, so you can add or remove registries before a test, and also check if your data was stored correctly.
```

Comment:
```suggestion
MUnit DB Server enables you to execute instructions to the in-memory databases so that you can add or remove registries before a test and verify that your data was stored correctly.
```